feat(source): adding the source page

boaz0 opened the pull request:
- [x] Work on the "remove source" dialog message
- [x] work on the "remove source" API - client side
- [x] Expand table item
- [x] Filter by name/type
- [x] Locales support
- [x] Selectable table item (commented)
- [x] Add states: no match found, Loading, Error and Empty

dlabrecq commented:
Please let me know when the PR is ready for a review. Unless you need some feedback now?

boaz0 commented:
I'm blocked now. I keep getting 404 when attempting to fetch sources or reports.

dlabrecq commented:
Try rebasing your code. Chris made some updates to the UI to accommodate Insights recent changes.

You need to point your browser at a new URL. See https://github.com/project-koku/koku-ui/pull/622

boaz0 commented:
//cc @dlabrecq waiting for your review. Thanks.

boaz0 commented on src/routes.tsx:
```diff
@@ -57,6 +61,13 @@ const routes: AppRoute[] = [
+    component: SourceDetails,
+    exact: true,
+    icon: MoneyBillIcon,
+  },
```
Nope, it's not working :sob:

boaz0 commented:
@dlabrecq - done. renamed the files/directories and changed the "Add Source" button in the Overview page to "Add Sources" that will go to the Source Setting page.

boaz0 commented:
Thanks a lot @dlabrecq
